client/src/pages/CreateProduct.js

```diff
@@ -199,6 +219,16 @@ export default function CreateProduct() {
                                     />
                                 </Paper>
                             ))}
+                        </div> */}
+                        <div className={classes.rootCard}>
+                            {["1", "2", "3", "4", "5", "6"].map(num => (
```

Comment:
```suggestion
                            {new Array(6).map(num => (
```

since num isn't really used and you're just looking to create an array of 6 elem